### PR TITLE
Fix metric docstring

### PR DESCRIPTION
**Pull request by AkshitaB**
Fixes https://github.com/allenai/allennlp/issues/5472

### CODE REVIEW

File: allennlp/training/metrics/fbeta_multi_label_measure.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ class FBetaMultiLabelMeasure(FBetaMeasure):
         in the data will result in 0 components in a macro or weighted average.
 
     threshold: `float`, optional (default = `0.5`)
-        Logits over this threshold will be considered predictions for the corresponding class.
+        Probabilities over this threshold will be considered predictions for the corresponding class.
+        Note that you can also use this metric with logits, by setting the `threshold` value to `0.0`.
```
Comment (epwalsh) [inline allennlp/training/metrics/fbeta_multi_label_measure.py:65]: How about:
```suggestion
        Note that you can also use this metric with logits, in which case it would make more sense to set
        the `threshold` value to `0.0`.
```

**Review by epwalsh — APPROVED** [review]
